internal/storage/db/internal/db.go

```diff
@@ -263,7 +264,7 @@ func (s *dbStorage) GetCompilationUnits(ctx context.Context, ids ...namer.Module
 		Where(
 			goqu.And(
 				goqu.C(PolicyDepTblPolicyIDCol).Table("pd").In(ids),
-				goqu.C(PolicyTblDisabledCol).Table("p").Eq(false),
+				goqu.C(PolicyTblDisabledCol).Table("p").Eq(DialectFalseValue(s.db.Dialect())),
```

Comment:
Any idea why the dialect doesn't handle this conversion automatically? It seems to be configured correctly: https://github.com/doug-martin/goqu/blob/31d438d38230118d377f11cfaad9627aa3a23864/dialect/sqlserver/sqlserver.go#L33

Reply:
Not sure why it wasn't being picked up. It works if we replace `false` with `goqu.V(false)`.